include/wtmpdb.h

```diff
@@ -35,14 +35,11 @@
 #define _VARLINK_WTMPDB_SOCKET_DIR "/run/wtmpdb"
 #define _VARLINK_WTMPDB_SOCKET _VARLINK_WTMPDB_SOCKET_DIR"/socket"
 
-#define EMPTY           0  /* No valid user accounting information.  */
-#define BOOT_TIME       1  /* Time of system boot.  */
-#define RUNLEVEL        2  /* The system's runlevel. Unused with systemd. */
-#define USER_PROCESS    3  /* Normal process.  */
+#define WTMPDB_EMPTY           0  /* No valid user accounting information.  */
+#define WTMPDB_BOOT_TIME       1  /* Time of system boot.  */
+#define WTMPDB_RUNLEVEL        2  /* The system's runlevel. Unused with systemd. */
+#define WTMPDB_USER_PROCESS    3  /* Normal process.  */
```

Comment:
This is a public interface used by applications calling libwtmpdb directly. Changing this would be an API change, which is a nogo.
You should have noticed that when there was the need to modify the tests to use an internal header file.